### PR TITLE
chore : setup stale action workflow

### PR DESCRIPTION
**Pull request by ajredniwja**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

1. To make sure we include your contribution in the release notes, please make sure to add description entry for your changes in the "unreleased changes" section of the `CHANGELOG.md` file (at corresponding gem). For the description entry, please make sure it lives in one line and starts with `Feature` or `Issue` in the correct format.

2. For generated code changes, please checkout below instructions first:
  https://github.com/aws/aws-sdk-ruby/blob/master/CONTRIBUTING.md

Thank you for your contribution!

### CODE REVIEW

File: .github/workflows/stale_issues.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        response-requested-label: response-requested
+
+        # Don't set closed-for-staleness label to skip closing very old issues
+        # regardless of label
+        closed-for-staleness-label: closed-for-staleness
+
+        # Issue timing
+        days-before-stale: 7
+        days-before-close: 4
+        days-before-ancient: 1095
```
Comment (mullermp) [inline .github/workflows/stale_issues.yml:34]: This is really ancient... why not 365?

File: .github/workflows/stale_issues.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+    name: Stale issue job
+    steps:
+    - uses: aws-actions/stale-issue-cleanup@v3
+      with:
+        # Setting messages to an empty string will cause the automation to skip
+        # that category
+        ancient-issue-message: Greetings!! We noticed that this issue is very old and has not been updated in a very long time. We will close this issue for now. Please feel free to reach out and open a new issue if you think this is still a relevant issue or if it is an error.
```
Comment (alextwoods) [inline .github/workflows/stale_issues.yml:17]: Is this the language we are using cross SDKs?  I'd lean towards removing some of the less matter of fact/precise language like "Greetings!!" (this is an automated response, two exclamation marks feels a little overly enthusiastic).  and "very old" (imprecise).  and "for now" (not straight forward - does this mean we have a process to re-open these?)

Comment (mullermp) [reply .github/workflows/stale_issues.yml:17]: I'm assuming it is cross SDK regardless..

Comment (mullermp) [reply .github/workflows/stale_issues.yml:17]: https://github.com/aws/aws-sdk-js/pull/3242

Comment (ajredniwja) [reply .github/workflows/stale_issues.yml:17]: This is not cross SDK

**Pull request merged**